packages/forms-web-app/src/views/having-your-say-guide/registering-have-your-say.njk

```diff
@@ -40,40 +40,52 @@
+
+                <h2 class="govuk-heading-m" id="what-you-need-to-register">What you need to register</h2>
+                <p class="govuk-body">To register, you will need to give us your:</p>
                 <ul class="govuk-list govuk-list--bullet">
                   <li>full name</li>
                   <li>address, email and telephone number</li>
-                  <li>your comments about the project and how it affects you</li>
+                  <li>your comments about the project</li>
                 </ul>
-                <p class="govuk-body">After you register, you will be given an interested party reference. The information you give us about how the project affects you will be published on our website.</p>
-                <p class="govuk-body">The developer must make the project public by advertising it and giving details about how you can register.</p>
-                <h2 class="govuk-heading-m" id="find-a-project-and-register">Find a project and register</h2>
-                <p class="govuk-body">The registration for each project will be open for at least 28 days to give you time to register. The deadline for registering will be in the developer's advert, or you can check the project page.
-                </p>
-                <p class="govuk-body"><a href="#">Find a project and register to have your say about a national infrastructure project</a></p>
+                <p class="govuk-body">After you register, you will be given an interested party reference. Your name and comments will be published on our website. Your address, email address and telephone number will be kept private. <a target="_blank" href="https://www.gov.uk/government/publications/planning-inspectorate-privacy-notices/customer-privacy-notice">You can view our privacy notice for more information (opens in a new tab)</a></p>
+                
+                <h2 class="govuk-heading-m" id="after-you-have-registered">After you have registered</h2>
+                <p class="govuk-body">Once you have registered to have your say, your name and comments will be published on the project section of this website. Once the registration period closes, you can see all of the other registration comments (also known as Relevant Representations) from other people.</p>
+                <p class="govuk-body">Once the Examination of the application starts, you will be able to submit more information at the deadlines in the timetable. You can do this using the online submission form in the specific project section of this website.</p>
+
+                <h2 class="govuk-heading-m" id="register-to-have-your-say">Register to have your say</h2>
+                <p class="govuk-body">The deadline for registering will be in the developer's advert, or you can check the project page.</p>
+                <p class="govuk-body"><a href="../examination/{{ caseRef }}">Go to the {{ projectName }} overview</a></p>
```

Comment:
I think caseRef and projectName are not available dynamically. So either requirement should be to hardcode both depending on which project they want.

Reply:
Also branch has conflicts.